src/commands/notify.yml

```diff
@@ -42,7 +42,12 @@ parameters:
       Select which channel in which to post to. Channel name or ID will work. You may include a comma separated list of channels if you wish to post to multiple channels at once. Set the "SLACK_DEFAULT_CHANNEL" environment variable for the default channel.
     type: string
     default: $SLACK_DEFAULT_CHANNEL
-
+  exit_on_fail:
```

Comment:
This name is a bit weird. The command will always exit whether the API request succeeds or fails. Suggestions:

* `ignore_failures` (inverted condition)
* `report_failures`

Reply:
Excellent suggestion

Reply:
@alexjurkiewicz How would you feel about `ignore_errors` as to not potentially confuse `failures` with the `fail` event and CI job failures?

Reply:
good point. Sounds perfect 👌